outlines/text/generate/api.py

```diff
@@ -17,13 +18,16 @@ def json(
         "The old import path will be removed in Outlines v0.1.0.",
         DeprecationWarning,
     )
-    return outlines.generate.json(model, schema_object, max_tokens, sampler=sampler)
+    return outlines.generate.json(
+        model, schema_object, max_tokens, stop, sampler=sampler
```

Comment:
The old interface used `stop_at`, we should keep that name at least here

Reply:
I put `stop` because it's what `continuation` seemed to expect on line 49 and what you removed in your big PR. Was it `stop_at` in an earlier version?

Reply:
Yes it was. `stop_at` sounds like a better name anyway, wdyt?

Reply:
Absolutely!